src/osa/paths.py

```diff
@@ -346,3 +347,23 @@ def create_source_directories(source_list: list, cuts_dir: Path):
         if source is not None:
             source_dir = cuts_dir / source
             source_dir.mkdir(parents=True, exist_ok=True)
+
+
+def get_latest_version_file(longterm_files: List[str]) -> Path:
+    """Get the latest version path of the produced longterm DL1 datacheck files for a given date."""
+    latest_version_path = max(longterm_files, key=lambda path: int(path.parents[1].name.split(".")[1]) if path.parents[1].name.startswith("v") else "")
+    return latest_version_path
+
+
+def create_longterm_symlink():
+    """If the created longterm DL1 datacheck file corresponds to the latest 
+    version available, make symlink to it in the "all" common directory."""
+    nightdir = date_to_dir(options.date)
+    longterm_dir = Path(cfg.get("LST1", "LONGTERM_DIR"))
+    linked_longterm_file = longterm_dir / f"night_wise/all/DL1_datacheck_{nightdir}.h5"
+    all_longterm_files = sorted(longterm_dir.rglob(f"v*/{nightdir}/DL1_datacheck_{nightdir}.h5"))
+    latest_version_file = get_latest_version_file(all_longterm_files)
+
+    log.info("Make symlink of the latest version longterm DL1 datacheck file in the common directory.")
+    linked_longterm_file.unlink()
```

Comment:
```suggestion
    linked_longterm_file.unlink(missing_ok=True)
```
I think this line for removing the existing link can be removed. Simply create the link, if it already exists, it will be overwritten.

>  If the target is present, the type of the symlink will be created to match. Otherwise, the symlink will be created as a directory if target_is_directory is True or a file symlink (the default) otherwise.